### PR TITLE
tpm_bind_luks: Update FS options written to crypttab and fstab

### PR DESCRIPTION
**Pull request by PaulYuuu**
It seems that a problem in the previous version, it causes the luks FS
sometimes not to mount after reboot, so correct the mount options in
crypttab and fstab. We should use UUID instead of disk path in crypttab 
to avoid the disk letter changes, also relabel the mount point is a must
to avoid another failure.

ID: 2168025
Signed-off-by: Yihuang Yu <yihyu@redhat.com>

### CODE REVIEW

**Comment by PaulYuuu** [comment]
Hello @qcheng-redhat, could you help to do a regression test?

**Comment by PaulYuuu** [comment]
```
 (1/1) Host_RHEL.m9.u2.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.aarch64.io-github-autotest-qemu.tpm_bind_luks.arm64-pci: PASS (147.81 s)
```

**Comment by qcheng-redhat** [comment]
The latest version looks good to me. 

With 5*2 runs, no failure.
 (1/5) repeat1.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: STARTED
 (1/5) repeat1.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: PASS (223.69 s)
 (2/5) repeat2.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: STARTED
 (2/5) repeat2.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: PASS (222.67 s)
 (3/5) repeat3.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: STARTED
 (3/5) repeat3.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: PASS (225.66 s)
 (4/5) repeat4.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: STARTED
 (4/5) repeat4.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: PASS (225.63 s)
 (5/5) repeat5.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: STARTED
 (5/5) repeat5.Host_RHEL.m8.u8.product_rhel.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.8.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: PASS (223.06 s)

**Comment by qcheng-redhat** [comment]
There are still some failures with more runs on another host, about 2/5 fail ratio. 

 (3/5) repeat3.Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: STARTED
 (3/5) repeat3.Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: ERROR: Cannot mount /dev/mapper/luks_fs to /mnt/luks_fs (378.89 s)
 (4/5) repeat4.Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: STARTED
 (4/5) repeat4.Host_RHEL.m9.u2.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.2.0.x86_64.io-github-autotest-qemu.tpm_bind_luks.q35: ERROR: Cannot mount /dev/mapper/luks_fs to /mnt/luks_fs (382.58 s)

**Comment by PaulYuuu** [comment]
@qcheng-redhat yes, seems related SELinux, or a bug. I am debugging.

**Comment by PaulYuuu** [comment]
Hello @qcheng-redhat, can you help to retest? The problem is related to the mount options, I updated it and tested it 50 times, and all passed.

File: qemu/tests/tpm_bind_luks.py [hunk 1]
```diff
@@ -53,7 +53,11 @@ def get_md5sum():
         """
         Return the file's MD5
         """
-        return session.cmd_output("md5sum " + dd_file).split()[0]
+        status, md5 = session.cmd_status_output(
+            "md5sum " + dd_file, print_func=test.log.info).split()[0]
```
Comment (qcheng-redhat) [inline qemu/tests/tpm_bind_luks.py:57]: Got error: ERROR: 'tuple' object has no attribute 'split'. No need split()[0].

Comment (PaulYuuu) [reply qemu/tests/tpm_bind_luks.py]: Thank you. Forgot to update this line, corrected it.

**Comment by PaulYuuu** [comment]
@qcheng-redhat， I think the problem is gone. The root cause is the disk letter changes, which causes TPM sometimes cannot to decrypt the correct disk after reboot the guest.

**Comment by qcheng-redhat** [comment]
LGTM

With 10*3 runs, no failures.

**Pull request merged**